lib/puma/launcher.rb

```diff
@@ -182,6 +182,9 @@ def refork
       if clustered? && @runner.respond_to?(:fork_worker!) && @options[:fork_worker]
         @runner.fork_worker!
         true
+      elsif clustered? && @runner.respond_to?(:mold_and_refork!) && @options[:mold_worker]
```

Comment:
Nit. I understand that if we're here there's a high probability that this was called intentionally, but can we please check the option before going through the trouble of checking if the method is available?

```suggestion
      elsif clustered? && @options[:mold_worker] && @runner.respond_to?(:mold_and_refork!)
```

Reply:
I think it's acceptable to update the `fork_worker` condition above as well.